os/fs/smartfs/smartfs.h

```diff
@@ -175,7 +175,7 @@
 #define INODE_STATE_FILE          (CONFIG_NXFFS_ERASEDSTATE ^ 0x22)
 #define INODE_STATE_DELETED       (CONFIG_NXFFS_ERASEDSTATE ^ 0xaa)
 
-/* Smartfs worbuffer maxuimum length */
+/* jSmartFs worbuffer maximum length */
```

Comment:
Hi @mkrunal , please follow the rules for the commit title format.

https://github.com/Samsung/TizenRT/wiki/Documentations#commit-rules